yellowbrick/features/importances.py

```diff
@@ -207,9 +246,27 @@ def finalize(self, **kwargs):
         # Remove the ygrid
         self.ax.grid(False, axis='y')
 
+        if self.stack:
+            plt.legend(bbox_to_anchor=(1.04, 0.5), loc="center left")
         # Ensure we have a tight fit
         plt.tight_layout()
 
+    def _find_classes_param(self):
+        """
+        Searches the wrapped model for the classes_ parameter.
+        """
+        for attr in ["classes_"]:
+            try:
+                return getattr(self.estimator, attr)
+            except AttributeError:
+                continue
+
+        raise YellowbrickTypeError(
```

Comment:
Technically, at this point, you could use `np.unique` since you're using the `is_classifier` check and you know this is a classifier. However, I think this is just fine and does inform the user about what's going on.